test/suite.js

```diff
@@ -1,15 +1,28 @@
 'use strict'
 
 var test = require('tape')
+var abstract = require('abstract-leveldown')
 
 function name () {
   var seq = name.seq = (name.seq || 0) + 1
   return 'level-test-' + seq
 }
 
-function innerDb (db) {
-  while (db.db) db = db.db
```

Comment:
The issue was that `db.db` is an IDBDatabase in `level-js`. To fix this I imported the `down` utility from `subleveldown`.